db/migrate/20180621051702_add_initial_setting_data.rb

```diff
@@ -0,0 +1,5 @@
+class AddInitialSettingData < ActiveRecord::Migration[5.1]
+  def up
```

Comment:
__[Reek]__

[UtilityFunction] AddInitialSettingData#up doesn't depend on instance state (maybe move it to another class?)


<!-- Comment by [Sider](https://sider.review) -->